cdragontoolbox/hashes.py

```diff
@@ -406,8 +439,8 @@ def grep_wad(self, wad):
 
             # relative path starting with ./ or ../ (e.g. require() use)
             relpaths |= {m.group(1) for m in re.finditer(r'[^a-zA-Z0-9/_.\\-]((?:\.|\.\.)/[a-zA-Z0-9/_.-]+)', data)}
-            # basename or subpath (check for a extension)
-            relpaths |= {m.group(1) for m in re.finditer(r'''["']([a-zA-Z0-9][a-zA-Z0-9/_.@-]*\.(?:js|json|webm|html|[a-z]{3}))\b''', data)}
+            # basename or subpath (check for an extension)
+            relpaths |= {m.group(1) for m in re.finditer(r'''["']([a-zA-Z0-9][a-zA-Z0-9/_.@-]*\.(?:js|json|webm|html|[a-z]{3}))["']''', data)}
```

Comment:
`\b` is more generic and handles more cases.

Reply:
Do you have an example for a file where neither ``"`` nor ``'`` is used for a relative path? Cause I could not imagine such a case and I feel like my change makes it logically more intuitive.

Reply:
I don't remember exactly where but I think I had such cases. You could imagine any separator at the end of the path.